cssom-view-1/Overview.bs

```diff
@@ -420,22 +420,25 @@ an associated element or [=pseudo-element=] <var>element</var> and optionally a
 (which is "<code>auto</code>" if omitted),
 the following steps must be run:
 
-<ol>
- <li><a lt="smooth scroll aborted">Abort</a> any ongoing <a>smooth scroll</a> for <var>box</var>.
- <li>If the user agent honors the 'scroll-behavior' property and one of the following are true:
+1. <a lt="smooth scroll aborted">Abort</a> any ongoing <a>smooth scroll</a> for <var>box</var>.
+1. Resolve any pending promise for <var>box</var>.
```

Comment:
This should be a bit more specific about what promises its resolving, e.g.

Resolve any pending scroll promises whose scroll container is <var>box</var>.

I'm not sure if we need to explicitly say that we store these in some set or if it can be implicit.